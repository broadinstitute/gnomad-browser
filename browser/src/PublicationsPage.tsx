import React from 'react'
import styled from 'styled-components'
import { ExternalLink, List, ListItem, PageHeading } from '@gnomad/ui'

import DocumentTitle from './DocumentTitle'
import InfoPage from './InfoPage'

const Citation = styled.cite`
  font-style: normal;
  line-height: 1.4;
`

export default () => (
  <InfoPage>
    <DocumentTitle title="Publications" />
    <PageHeading>Publications</PageHeading>
    <p>This page lists publications from the gnomAD group.</p>
    <p>
      If you want to cite gnomAD in a publication, please use the current flagship paper (except if
      referring to a previous version; see below). If you build online resources that include the
      data set, we ask that you provide links to the browser from the resource where relevant.
    </p>
    <p>
      There is no need to include us as authors on your manuscript, unless we contributed specific
      advice or analysis for your work.
    </p>
    <p>
      Current flagship paper (<b>v3</b>):
      {/* @ts-expect-error */}
      <List>
        {/* @ts-expect-error */}
        <ListItem>
          <Citation>
            Chen, S.*, Francioli, L. C.*, Goodrich, J. K., Collins, R. L., Kanai, M., Wang, Q.,
            Alföldi, J., Watts, N. A., Vittal, C., Gauthier, L. D., Poterba, T., Wilson, M. W.,
            Tarasova, Y., Phu, W., Grant, R., Yohannes, M. T., Koenig, Z., Farjoun, Y., Banks, E.,
            Donnelly, S., Gabriel, S., Gupta, N., Ferriera, S., Tolonen, C., Novod, S., Bergelson,
            L., Roazen, D., Ruano-Rubio, V., Covarrubias, M., Llanwarne, C., Petrillo, N., Wade, G.,
            Jeandet, T., Munshi, R., Tibbetts, K., Genome Aggregation Database (gnomAD) Consortium,
            O’Donnell-Luria, A., Solomonson, M., Seed, C., Martin, A. R., Talkowski, M. E., Rehm, H.
            L., Daly, M. J., Tiao, G., Neale, B. M.†, MacArthur, D. G.† & Karczewski, K. J. A
            genomic mutational constraint map using variation in 76,156 human genomes{' '}
            <em>Nature</em> 625, 92–100 (2024). {/* @ts-expect-error */}
            <ExternalLink href="https://doi.org/10.1038/s41586-023-06045-0">
              https://doi.org/10.1038/s41586-023-06045-0
            </ExternalLink>{' '}
            {/* @ts-expect-error */}
            <ExternalLink href="https://citation-needed.springer.com/v2/references/10.1038/s41586-023-06045-0?format=refman&flavour=citation">
              Download citation
            </ExternalLink>
          </Citation>
        </ListItem>
        * contributed equally
        <br />† contributed equally
      </List>
    </p>{' '}
    <p>
      Previous flagship papers:
      {/* @ts-expect-error */}
      <List>
        {/* @ts-expect-error */}
        <ListItem>
          <b>v2</b>:{' '}
          <Citation>
            Karczewski, K. J., Francioli, L. C., Tiao, G., Cummings, B. B., Alföldi, J., Wang, Q.,
            Collins, R. L., Laricchia, K. M., Ganna, A., Birnbaum, D. P., Gauthier, L. D., Brand,
            H., Solomonson, M., Watts, N. A., Rhodes, D., Singer-Berk, M., England, E. M., Seaby, E.
            G., Kosmicki, J. A., … MacArthur, D. G. The mutational constraint spectrum quantified
            from variation in 141,456 humans. <em>Nature</em> 581, 434–443 (2020).{' '}
            {/* @ts-expect-error */}
            <ExternalLink href="https://doi.org/10.1038/s41586-020-2308-7">
              https://doi.org/10.1038/s41586-020-2308-7
            </ExternalLink>{' '}
            {/* @ts-expect-error */}
            <ExternalLink href="https://citation-needed.springer.com/v2/references/10.1038/s41586-020-2308-7?format=refman&flavour=citation">
              Download citation
            </ExternalLink>
          </Citation>
        </ListItem>
        {/* @ts-expect-error */}
        <ListItem>
          <b>ExAC</b>:{' '}
          <Citation>
            Lek, M., Karczewski, K. J.*, Minikel, E. V.*, Samocha, K. E.*, Banks, E., Fennell, T.,
            O’Donnell-Luria, A. H., Ware, J. S., Hill, A. J., Cummings, B. B., Tukiainen, T.,
            Birnbaum, D. P., Kosmicki, J. A., Duncan, L. E., Estrada, K., Zhao, F., Zou, J.,
            Pierce-Hoffman, E., … Daly, M. J., MacArthur, D. G. & Exome Aggregation Consortium.
            Analysis of protein-coding genetic variation in 60,706 humans. <em>Nature</em> 536,
            285–291 (2016). {/* @ts-expect-error */}
            <ExternalLink href="https://doi.org/10.1038/nature19057">
              https://doi.org/10.1038/nature19057
            </ExternalLink>{' '}
            {/* @ts-expect-error */}
            <ExternalLink href="https://citation-needed.springer.com/v2/references/10.1038/nature19057?format=refman&flavour=citation">
              Download citation
            </ExternalLink>
          </Citation>
        </ListItem>
        * contributed equally
      </List>
    </p>
    <p>
      Remaining publications:{/* @ts-expect-error */}
      <List>
        {/* @ts-expect-error */}
        <ListItem>
          <Citation>
            Guo, M. H.†, Francioli, L. C.†, Stenton, S. L., Goodrich, J. K., Watts, N. A.,
            Singer-Berk, M., Groopman, E., Darnowsky, P. W., Solomonson, M., Baxter, S., gnomAD
            Project Consortium, Tiao, G., Neale, B. M., Hirschhorn, J. N., Rehm, H., Daly, M. J.,
            O’Donnell-Luria, A., Karczewski, K., MacArthur, D. G., Samocha, K. E. Inferring compound
            heterozygosity from large-scale exome sequencing data. <em>Nature Genetics</em> (2023).{' '}
            {/* @ts-expect-error */}
            <ExternalLink href="https://doi.org/10.1038/s41588-023-01608-3">
              https://doi.org/10.1038/s41588-023-01608-3
            </ExternalLink>
          </Citation>
        </ListItem>
        {/* @ts-expect-error */}
        <ListItem>
          <Citation>
            Gudmundsson, S., Singer-Berk, M., Watts, N. A., Phu, W., Goodrich, J. K., Solomonson,
            M., Genome Aggregation Database (gnomAD) Consortium, Rehm, H. L., MacArthur, D. G.,
            O’Donnell-Luria, A. Variant interpretation using population databases: Lessons from
            gnomAD. <em>Human Mutation</em> 1-19 (2021). {/* @ts-expect-error */}
            <ExternalLink href="https://doi.org/10.1002/humu.24309">
              https://doi.org/10.1002/humu.24309
            </ExternalLink>
          </Citation>
        </ListItem>
        {/* @ts-expect-error */}
        <ListItem>
          <Citation>
            Laricchia, K. M.*, Lake, N. J.*, Watts, N. A., Shand, M., Haessly, A., Gauthier, L. D.,
            Benjamin, D., Banks, E., Soto, J., Garimella, K., Emery, J., Genome Aggregation Database
            (gnomAD) Consortium, Rehm, H. L., MacArthur, D. G., Tiao, G.†, Lek, M.†, Mootha, V. K.†,
            Calvo, S. E.† Mitochondrial DNA variation across 56,434 individuals in gnomAD.{' '}
            <em>Genome Res.</em> 32, 569–582 (2022). {/* @ts-expect-error */}
            <ExternalLink href="https://doi.org/10.1101/gr.276013.121">
              https://doi.org/10.1101/gr.276013.121
            </ExternalLink>
          </Citation>
        </ListItem>
        {/* @ts-expect-error */}
        <ListItem>
          <Citation>
            Collins, R. L.*, Brand, H.*, Karczewski, K. J., Zhao, X., Alföldi, J., Francioli, L. C.,
            Khera, A. V., Lowther, C., Gauthier, L. D., Wang, H., Watts, N. A., Solomonson, M.,
            O’Donnell-Luria, A., Baumann, A., Munshi, R., Walker, M., Whelan, C., Huang, Y.,
            Brookings, T., … Talkowski, M. E. A structural variation reference for medical and
            population genetics. <em>Nature</em> 581, 444–451 (2020). {/* @ts-expect-error */}
            <ExternalLink href="https://doi.org/10.1038/s41586-020-2287-8">
              https://doi.org/10.1038/s41586-020-2287-8
            </ExternalLink>
          </Citation>
        </ListItem>
        {/* @ts-expect-error */}
        <ListItem>
          <Citation>
            Cummings, B. B., Karczewski, K. J., Kosmicki, J. A., Seaby, E. G., Watts, N. A.,
            Singer-Berk, M., Mudge, J. M., Karjalainen, J., Kyle Satterstrom, F., O’Donnell-Luria,
            A., Poterba, T., Seed, C., Solomonson, M., Alföldi, J., The Genome Aggregation Database
            Production Team, The Genome Aggregation Database Consortium, Daly, M. J., & MacArthur,
            D. G. Transcript expression-aware annotation improves rare variant interpretation.{' '}
            <em>Nature</em> 581, 452–458 (2020). {/* @ts-expect-error */}
            <ExternalLink href="https://doi.org/10.1038/s41586-020-2329-2">
              https://doi.org/10.1038/s41586-020-2329-2
            </ExternalLink>
          </Citation>
        </ListItem>
        {/* @ts-expect-error */}
        <ListItem>
          <Citation>
            Minikel, E. V., Karczewski, K. J., Martin, H. C., Cummings, B. B., Whiffin, N., Rhodes,
            D., Alföldi, J., Trembath, R. C., van Heel, D. A., Daly, M. J., Genome Aggregation
            Database Production Team, Genome Aggregation Database Consortium, Schreiber, S. L., &
            MacArthur, D. G. Evaluating potential drug targets through human loss-of-function
            genetic variation. <em>Nature</em> 581, 459–464 (2020). {/* @ts-expect-error */}
            <ExternalLink href="https://doi.org/10.1038/s41586-020-2267-z">
              https://doi.org/10.1038/s41586-020-2267-z
            </ExternalLink>
          </Citation>
        </ListItem>
        {/* @ts-expect-error */}
        <ListItem>
          <Citation>
            Wang, Q., Pierce-Hoffman, E., Cummings, B. B., Karczewski, K. J., Alföldi, J.,
            Francioli, L. C., Gauthier, L. D., Hill, A. J., O’Donnell-Luria, A. H., Genome
            Aggregation Database (gnomAD) Production Team, Genome Aggregation Database (gnomAD)
            Consortium, & MacArthur, D. G. Landscape of multi-nucleotide variants in 125,748 human
            exomes and 15,708 genomes. <em>Nature Communications</em> 11, 2539 (2020).{' '}
            {/* @ts-expect-error */}
            <ExternalLink href="https://doi.org/10.1038/s41467-019-12438-5">
              https://doi.org/10.1038/s41467-019-12438-5
            </ExternalLink>
          </Citation>
        </ListItem>
        {/* @ts-expect-error */}
        <ListItem>
          <Citation>
            Whiffin, N.*, Armean, I. M.*, Kleinman, A.*, Marshall, J. L., Minikel, E. V., Goodrich,
            J. K., Quaife, N. M., Cole, J. B., Wang, Q., Karczewski, K. J., Cummings, B. B.,
            Francioli, L., Laricchia, K., Guan, A., Alipanahi, B., Morrison, P., Baptista, M. A. S.,
            Merchant, K. M., Genome Aggregation Database Production Team, … Cannon, P.†, MacArthur,
            D. G.† The effect of LRRK2 loss-of-function variants in humans. <em>Nature Medicine</em>{' '}
            (2020). {/* @ts-expect-error */}
            <ExternalLink href="https://doi.org/10.1038/s41591-020-0893-5">
              https://doi.org/10.1038/s41591-020-0893-5
            </ExternalLink>
          </Citation>
        </ListItem>
        {/* @ts-expect-error */}
        <ListItem>
          <Citation>
            Whiffin, N., Karczewski, K. J., Zhang, X., Chothani, S., Smith, M. J., Gareth Evans, D.,
            Roberts, A. M., Quaife, N. M., Schafer, S., Rackham, O., Alföldi, J., O’Donnell-Luria,
            A. H., Francioli, L. C., Genome Aggregation Database (gnomAD) Production Team, Genome
            Aggregation Database (gnomAD) Consortium, Cook, S. A., Barton, P. J. R., MacArthur, D.
            G.†, & Ware, J. S.† Characterising the loss-of-function impact of 5’ untranslated region
            variants in 15,708 individuals. <em>Nature Communications</em> 11, 2523 (2020).{' '}
            {/* @ts-expect-error */}
            <ExternalLink href="https://doi.org/10.1038/s41467-019-10717-9">
              https://doi.org/10.1038/s41467-019-10717-9
            </ExternalLink>
          </Citation>
        </ListItem>
        {/* @ts-expect-error */}
        <ListItem>
          <Citation>
            Karczewski, K. J., Gauthier, L. D., & Daly, M. J. Technical artifact drives apparent
            deviation from Hardy-Weinberg equilibrium at CCR5-∆32 and other variants in gnomAD{' '}
            <em>bioRxiv</em> (p. 784157). {/* @ts-expect-error */}
            <ExternalLink href="https://doi.org/10.1101/784157">
              https://doi.org/10.1101/784157
            </ExternalLink>
          </Citation>
        </ListItem>
        {/* @ts-expect-error */}
        <ListItem>
          <Citation>
            Karczewski, K. J., Weisburd, B., Thomas, B., Solomonson, M., Ruderfer, D. M., Kavanagh,
            D., Hamamsy, T., Lek, M., Samocha, K. E., Cummings, B. B., Birnbaum, D., The Exome
            Aggregation Consortium, Daly, M. J., MacArthur, D. G.. The ExAC browser: displaying
            reference data information from over 60000 exomes. <em>Nucleic Acids Research</em>,
            Volume 45, Issue D1, January 2017, Pages D840–D845, {/* @ts-expect-error */}
            <ExternalLink href="https://doi.org/10.1093/nar/gkw971">
              https://doi.org/10.1093/nar/gkw971
            </ExternalLink>
          </Citation>
        </ListItem>
        * contributed equally
        <br />† contributed equally
      </List>
    </p>
  </InfoPage>
)
