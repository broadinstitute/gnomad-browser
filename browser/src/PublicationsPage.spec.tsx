import { expect, test } from '@jest/globals'
import 'jest-styled-components'

import React from 'react'
import renderer from 'react-test-renderer'

import PublicationsPage from './PublicationsPage'

import { BrowserRouter } from 'react-router-dom'

test('Publications Page has no unexpected changes', () => {
  const tree = renderer.create(
    <BrowserRouter>
      <PublicationsPage />
    </BrowserRouter>
  )
  expect(tree).toMatchSnapshot()
})
