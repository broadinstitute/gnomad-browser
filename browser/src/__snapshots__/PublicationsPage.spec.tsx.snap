// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Publications Page has no unexpected changes 1`] = `
.c6 {
  color: #1173bb;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c6:visited,
.c6:active {
  color: #1173bb;
}

.c6:focus,
.c6:hover {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.c3 {
  padding-left: 20px;
  list-style-type: disc;
  margin-bottom: 1em;
}

.c4 {
  margin-bottom: 0.5em;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  padding-bottom: 0.5em;
  border-bottom: 1px solid #ccc;
  margin: 0.67em 0;
  font-size: 14px;
}

.c2 {
  margin: 0;
}

.c0 {
  box-sizing: border-box;
  width: 100%;
  max-width: 1200px;
  padding: 0 15px;
  margin: 0 auto 40px;
  font-size: 14px;
  font-size: 16px;
}

.c0 p {
  margin-bottom: 1em;
  line-height: 1.4;
}

.c5 {
  font-style: normal;
  line-height: 1.4;
}

@media (max-width:900px) {
  .c1 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    border-bottom: none;
  }
}

@media (max-width:900px) {
  .c2 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    padding-bottom: 0.5em;
    border-bottom: 1px solid #ccc;
    margin-bottom: 0.5em;
  }
}

<div
  className="c0"
>
  <div
    className="c1"
  >
    <h1
      className="c2"
    >
      Publications
    </h1>
  </div>
  <p>
    This page lists publications from the gnomAD group.
  </p>
  <p>
    If you want to cite gnomAD in a publication, please use the current flagship paper (except if referring to a previous version; see below). If you build online resources that include the data set, we ask that you provide links to the browser from the resource where relevant.
  </p>
  <p>
    There is no need to include us as authors on your manuscript, unless we contributed specific advice or analysis for your work.
  </p>
  <p>
    Current flagship paper (
    <b>
      v3
    </b>
    ):
    <ul
      className="c3"
    >
      <li
        className="c4"
      >
        <cite
          className="c5"
        >
          Chen, S.*, Francioli, L. C.*, Goodrich, J. K., Collins, R. L., Kanai, M., Wang, Q., Alföldi, J., Watts, N. A., Vittal, C., Gauthier, L. D., Poterba, T., Wilson, M. W., Tarasova, Y., Phu, W., Grant, R., Yohannes, M. T., Koenig, Z., Farjoun, Y., Banks, E., Donnelly, S., Gabriel, S., Gupta, N., Ferriera, S., Tolonen, C., Novod, S., Bergelson, L., Roazen, D., Ruano-Rubio, V., Covarrubias, M., Llanwarne, C., Petrillo, N., Wade, G., Jeandet, T., Munshi, R., Tibbetts, K., Genome Aggregation Database (gnomAD) Consortium, O’Donnell-Luria, A., Solomonson, M., Seed, C., Martin, A. R., Talkowski, M. E., Rehm, H. L., Daly, M. J., Tiao, G., Neale, B. M.†, MacArthur, D. G.† & Karczewski, K. J. A genomic mutational constraint map using variation in 76,156 human genomes
           
          <em>
            Nature
          </em>
           625, 92–100 (2024). 
          <a
            className="c6"
            href="https://doi.org/10.1038/s41586-023-06045-0"
            rel="noopener noreferrer"
            target="_blank"
          >
            https://doi.org/10.1038/s41586-023-06045-0
          </a>
           
          <a
            className="c6"
            href="https://citation-needed.springer.com/v2/references/10.1038/s41586-023-06045-0?format=refman&flavour=citation"
            rel="noopener noreferrer"
            target="_blank"
          >
            Download citation
          </a>
        </cite>
      </li>
      * contributed equally
      <br />
      † contributed equally
    </ul>
  </p>
   
  <p>
    Previous flagship papers:
    <ul
      className="c3"
    >
      <li
        className="c4"
      >
        <b>
          v2
        </b>
        :
         
        <cite
          className="c5"
        >
          Karczewski, K. J., Francioli, L. C., Tiao, G., Cummings, B. B., Alföldi, J., Wang, Q., Collins, R. L., Laricchia, K. M., Ganna, A., Birnbaum, D. P., Gauthier, L. D., Brand, H., Solomonson, M., Watts, N. A., Rhodes, D., Singer-Berk, M., England, E. M., Seaby, E. G., Kosmicki, J. A., … MacArthur, D. G. The mutational constraint spectrum quantified from variation in 141,456 humans. 
          <em>
            Nature
          </em>
           581, 434–443 (2020).
           
          <a
            className="c6"
            href="https://doi.org/10.1038/s41586-020-2308-7"
            rel="noopener noreferrer"
            target="_blank"
          >
            https://doi.org/10.1038/s41586-020-2308-7
          </a>
           
          <a
            className="c6"
            href="https://citation-needed.springer.com/v2/references/10.1038/s41586-020-2308-7?format=refman&flavour=citation"
            rel="noopener noreferrer"
            target="_blank"
          >
            Download citation
          </a>
        </cite>
      </li>
      <li
        className="c4"
      >
        <b>
          ExAC
        </b>
        :
         
        <cite
          className="c5"
        >
          Lek, M., Karczewski, K. J.*, Minikel, E. V.*, Samocha, K. E.*, Banks, E., Fennell, T., O’Donnell-Luria, A. H., Ware, J. S., Hill, A. J., Cummings, B. B., Tukiainen, T., Birnbaum, D. P., Kosmicki, J. A., Duncan, L. E., Estrada, K., Zhao, F., Zou, J., Pierce-Hoffman, E., … Daly, M. J., MacArthur, D. G. & Exome Aggregation Consortium. Analysis of protein-coding genetic variation in 60,706 humans. 
          <em>
            Nature
          </em>
           536, 285–291 (2016). 
          <a
            className="c6"
            href="https://doi.org/10.1038/nature19057"
            rel="noopener noreferrer"
            target="_blank"
          >
            https://doi.org/10.1038/nature19057
          </a>
           
          <a
            className="c6"
            href="https://citation-needed.springer.com/v2/references/10.1038/nature19057?format=refman&flavour=citation"
            rel="noopener noreferrer"
            target="_blank"
          >
            Download citation
          </a>
        </cite>
      </li>
      * contributed equally
    </ul>
  </p>
  <p>
    Remaining publications:
    <ul
      className="c3"
    >
      <li
        className="c4"
      >
        <cite
          className="c5"
        >
          Guo, M. H.†, Francioli, L. C.†, Stenton, S. L., Goodrich, J. K., Watts, N. A., Singer-Berk, M., Groopman, E., Darnowsky, P. W., Solomonson, M., Baxter, S., gnomAD Project Consortium, Tiao, G., Neale, B. M., Hirschhorn, J. N., Rehm, H., Daly, M. J., O’Donnell-Luria, A., Karczewski, K., MacArthur, D. G., Samocha, K. E. Inferring compound heterozygosity from large-scale exome sequencing data. 
          <em>
            Nature Genetics
          </em>
           (2023).
           
          <a
            className="c6"
            href="https://doi.org/10.1038/s41588-023-01608-3"
            rel="noopener noreferrer"
            target="_blank"
          >
            https://doi.org/10.1038/s41588-023-01608-3
          </a>
        </cite>
      </li>
      <li
        className="c4"
      >
        <cite
          className="c5"
        >
          Gudmundsson, S., Singer-Berk, M., Watts, N. A., Phu, W., Goodrich, J. K., Solomonson, M., Genome Aggregation Database (gnomAD) Consortium, Rehm, H. L., MacArthur, D. G., O’Donnell-Luria, A. Variant interpretation using population databases: Lessons from gnomAD. 
          <em>
            Human Mutation
          </em>
           1-19 (2021). 
          <a
            className="c6"
            href="https://doi.org/10.1002/humu.24309"
            rel="noopener noreferrer"
            target="_blank"
          >
            https://doi.org/10.1002/humu.24309
          </a>
        </cite>
      </li>
      <li
        className="c4"
      >
        <cite
          className="c5"
        >
          Laricchia, K. M.*, Lake, N. J.*, Watts, N. A., Shand, M., Haessly, A., Gauthier, L. D., Benjamin, D., Banks, E., Soto, J., Garimella, K., Emery, J., Genome Aggregation Database (gnomAD) Consortium, Rehm, H. L., MacArthur, D. G., Tiao, G.†, Lek, M.†, Mootha, V. K.†, Calvo, S. E.† Mitochondrial DNA variation across 56,434 individuals in gnomAD.
           
          <em>
            Genome Res.
          </em>
           32, 569–582 (2022). 
          <a
            className="c6"
            href="https://doi.org/10.1101/gr.276013.121"
            rel="noopener noreferrer"
            target="_blank"
          >
            https://doi.org/10.1101/gr.276013.121
          </a>
        </cite>
      </li>
      <li
        className="c4"
      >
        <cite
          className="c5"
        >
          Collins, R. L.*, Brand, H.*, Karczewski, K. J., Zhao, X., Alföldi, J., Francioli, L. C., Khera, A. V., Lowther, C., Gauthier, L. D., Wang, H., Watts, N. A., Solomonson, M., O’Donnell-Luria, A., Baumann, A., Munshi, R., Walker, M., Whelan, C., Huang, Y., Brookings, T., … Talkowski, M. E. A structural variation reference for medical and population genetics. 
          <em>
            Nature
          </em>
           581, 444–451 (2020). 
          <a
            className="c6"
            href="https://doi.org/10.1038/s41586-020-2287-8"
            rel="noopener noreferrer"
            target="_blank"
          >
            https://doi.org/10.1038/s41586-020-2287-8
          </a>
        </cite>
      </li>
      <li
        className="c4"
      >
        <cite
          className="c5"
        >
          Cummings, B. B., Karczewski, K. J., Kosmicki, J. A., Seaby, E. G., Watts, N. A., Singer-Berk, M., Mudge, J. M., Karjalainen, J., Kyle Satterstrom, F., O’Donnell-Luria, A., Poterba, T., Seed, C., Solomonson, M., Alföldi, J., The Genome Aggregation Database Production Team, The Genome Aggregation Database Consortium, Daly, M. J., & MacArthur, D. G. Transcript expression-aware annotation improves rare variant interpretation.
           
          <em>
            Nature
          </em>
           581, 452–458 (2020). 
          <a
            className="c6"
            href="https://doi.org/10.1038/s41586-020-2329-2"
            rel="noopener noreferrer"
            target="_blank"
          >
            https://doi.org/10.1038/s41586-020-2329-2
          </a>
        </cite>
      </li>
      <li
        className="c4"
      >
        <cite
          className="c5"
        >
          Minikel, E. V., Karczewski, K. J., Martin, H. C., Cummings, B. B., Whiffin, N., Rhodes, D., Alföldi, J., Trembath, R. C., van Heel, D. A., Daly, M. J., Genome Aggregation Database Production Team, Genome Aggregation Database Consortium, Schreiber, S. L., & MacArthur, D. G. Evaluating potential drug targets through human loss-of-function genetic variation. 
          <em>
            Nature
          </em>
           581, 459–464 (2020). 
          <a
            className="c6"
            href="https://doi.org/10.1038/s41586-020-2267-z"
            rel="noopener noreferrer"
            target="_blank"
          >
            https://doi.org/10.1038/s41586-020-2267-z
          </a>
        </cite>
      </li>
      <li
        className="c4"
      >
        <cite
          className="c5"
        >
          Wang, Q., Pierce-Hoffman, E., Cummings, B. B., Karczewski, K. J., Alföldi, J., Francioli, L. C., Gauthier, L. D., Hill, A. J., O’Donnell-Luria, A. H., Genome Aggregation Database (gnomAD) Production Team, Genome Aggregation Database (gnomAD) Consortium, & MacArthur, D. G. Landscape of multi-nucleotide variants in 125,748 human exomes and 15,708 genomes. 
          <em>
            Nature Communications
          </em>
           11, 2539 (2020).
           
          <a
            className="c6"
            href="https://doi.org/10.1038/s41467-019-12438-5"
            rel="noopener noreferrer"
            target="_blank"
          >
            https://doi.org/10.1038/s41467-019-12438-5
          </a>
        </cite>
      </li>
      <li
        className="c4"
      >
        <cite
          className="c5"
        >
          Whiffin, N.*, Armean, I. M.*, Kleinman, A.*, Marshall, J. L., Minikel, E. V., Goodrich, J. K., Quaife, N. M., Cole, J. B., Wang, Q., Karczewski, K. J., Cummings, B. B., Francioli, L., Laricchia, K., Guan, A., Alipanahi, B., Morrison, P., Baptista, M. A. S., Merchant, K. M., Genome Aggregation Database Production Team, … Cannon, P.†, MacArthur, D. G.† The effect of LRRK2 loss-of-function variants in humans. 
          <em>
            Nature Medicine
          </em>
           
          (2020). 
          <a
            className="c6"
            href="https://doi.org/10.1038/s41591-020-0893-5"
            rel="noopener noreferrer"
            target="_blank"
          >
            https://doi.org/10.1038/s41591-020-0893-5
          </a>
        </cite>
      </li>
      <li
        className="c4"
      >
        <cite
          className="c5"
        >
          Whiffin, N., Karczewski, K. J., Zhang, X., Chothani, S., Smith, M. J., Gareth Evans, D., Roberts, A. M., Quaife, N. M., Schafer, S., Rackham, O., Alföldi, J., O’Donnell-Luria, A. H., Francioli, L. C., Genome Aggregation Database (gnomAD) Production Team, Genome Aggregation Database (gnomAD) Consortium, Cook, S. A., Barton, P. J. R., MacArthur, D. G.†, & Ware, J. S.† Characterising the loss-of-function impact of 5’ untranslated region variants in 15,708 individuals. 
          <em>
            Nature Communications
          </em>
           11, 2523 (2020).
           
          <a
            className="c6"
            href="https://doi.org/10.1038/s41467-019-10717-9"
            rel="noopener noreferrer"
            target="_blank"
          >
            https://doi.org/10.1038/s41467-019-10717-9
          </a>
        </cite>
      </li>
      <li
        className="c4"
      >
        <cite
          className="c5"
        >
          Karczewski, K. J., Gauthier, L. D., & Daly, M. J. Technical artifact drives apparent deviation from Hardy-Weinberg equilibrium at CCR5-∆32 and other variants in gnomAD
           
          <em>
            bioRxiv
          </em>
           (p. 784157). 
          <a
            className="c6"
            href="https://doi.org/10.1101/784157"
            rel="noopener noreferrer"
            target="_blank"
          >
            https://doi.org/10.1101/784157
          </a>
        </cite>
      </li>
      <li
        className="c4"
      >
        <cite
          className="c5"
        >
          Karczewski, K. J., Weisburd, B., Thomas, B., Solomonson, M., Ruderfer, D. M., Kavanagh, D., Hamamsy, T., Lek, M., Samocha, K. E., Cummings, B. B., Birnbaum, D., The Exome Aggregation Consortium, Daly, M. J., MacArthur, D. G.. The ExAC browser: displaying reference data information from over 60000 exomes. 
          <em>
            Nucleic Acids Research
          </em>
          , Volume 45, Issue D1, January 2017, Pages D840–D845, 
          <a
            className="c6"
            href="https://doi.org/10.1093/nar/gkw971"
            rel="noopener noreferrer"
            target="_blank"
          >
            https://doi.org/10.1093/nar/gkw971
          </a>
        </cite>
      </li>
      * contributed equally
      <br />
      † contributed equally
    </ul>
  </p>
</div>
`;
